js/common/lib/tensor.ts

```diff
@@ -335,12 +335,12 @@ export interface TensorFactory {
   /**
    * create a tensor from image object - HTMLImageElement, ImageData, ImageBitmap, URL
    *
-   * @param url - {string} - Assuming the string is a URL to an image
+   * @param string - {string} - Assuming the string is a URL to an image or Data URL
```

Comment:
urlSource

Reply:
Thanks for catching this - I will update the comment